test/Microsoft.Framework.Runtime.Tests/ProjectFacts.cs

```diff
@@ -373,7 +373,7 @@ public void CompilerOptionsAreSet()
         {
             var project = Project.GetProject(@"
 {
-    ""compilationOptions"": { ""allowUnsafe"": true, ""define"": [""X"", ""y""], ""platform"": ""x86"", ""warningsAsErrors"": true, ""optimize"": true }
+    ""compilationOptions"": { ""allowUnsafe"": true, ""define"": [""X"", ""y""], ""platform"": ""x86"", ""warningsAsErrors"": true, ""optimize"": true, ""strongNameKeyFile"" : ""c:\\keyfile.snk"", ""delaySign"" : true }
```

Comment:
Not sure if I get the purpose - wouldn't it be just like testing the JSON parser (which I assume is already tested)? We only allow `true/false` `bool` values in the project.json and I assume `rawOptions.ValueAsNullableBoolean("delaySign")` will break on everything else. I left `1`/`true` for environment variables to not break anyone who is already using it (e.g. NuGet).